consent-ws/src/main/java/org/broadinstitute/consent/http/resources/ConsentResource.java

```diff
@@ -196,4 +197,12 @@ private void checkValidDUL(Consent rec) {
         }
     }
 
+    private void checkConsentElection(String consentId) throws Exception {
+        Consent consent = populateFromApi(consentId);
+        Boolean consentElectionArchived = consent.getLastElectionArchived();
+
+        if (consentElectionArchived != null && !consentElectionArchived) {
+            throw new UpdateConsentException(String.format("Consent with Id '%s' can't be updated.", consentId));
```

Comment:
Nit: Language suggestion for the error message: 
```
"Consent '%s' cannot be updated with an active election."
```

Reply:
Are we considering "active" as Open or NOT archived?

Reply:
As you stated in a previous comment, consents can be updated when:
1. there have been no previous elections created for this consent
2. the most recent election has been closed/cancelled AND archived
If you want to suggest an error message we'll be glad to change it.

Reply:
Thanks Walter - to be clear, I'm offering suggestions that should be approved by the rest of the team.